### PR TITLE
164796898 Implement user add tags endpoint

### PR DESCRIPTION
**Pull request by toluola**
#### Description
- A user will be able to add tags to their articles 

#### Type of change
-  New feature (a non-breaking change which adds functionality)

#### How Has This Been Tested?
- Integration
- unit

#### Checklist:
- [x] my code follows style guidelines for this project

#### PT
- [#164796898](https://www.pivotaltracker.com/story/show/164796898)

### CODE REVIEW

File: src/controllers/article_tag.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Article } from '../db_api/models/index';
```
Comment (chikeozulumba) [inline src/controllers/article_tag.js]: @toluola Naming convention of the file is not conventional with the team. Kindly review

File: src/controllers/article_tag.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const articleTag = async (req, res, next) => {
```
Comment (chikeozulumba) [inline src/controllers/article_tag.js]: @toluola Nice work done, but you can restructure your code better, make use of destructuring and other modern conventions to improve your code quality.

File: src/routers/article_tag.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
```
Comment (chikeozulumba) [inline src/routers/article_tag.js]: @toluola Kindly refer to @Temmyogunbo suggested routing convention

File: src/routers/index.js [hunk 4]
```diff
@@ -1,7 +1,10 @@
 import { Router } from 'express';
+import articlesTag from './article_tag';
 
 const router = Router();
 
+router.use('/article/tags', articlesTag);
```
Comment (chikeozulumba) [inline src/routers/index.js]: @toluola same for the above

**Review by chikeozulumba — DISMISSED** [review]
@toluola Kindly review my comments, nice job done

File: src/routers/index.js (hunk 4 above)
Comment (toluola) [reply src/routers/index.js]: All has been fixed. Thanks

File: src/routers/article_tag.js (hunk 3 above)
Comment (toluola) [reply src/routers/article_tag.js]: fixed

File: src/controllers/article_tag.js (hunk 2 above)
Comment (toluola) [reply src/controllers/article_tag.js]: fixed

File: src/controllers/article_tag.js (hunk 1 above)
Comment (toluola) [reply src/controllers/article_tag.js]: fixed

**Comment by chikeozulumba** [comment]
@toluola So many merge conflicts, could you refactor

**Pull request closed without merging**